pkg/registry/registry.go

```
@@ -0,0 +1,78 @@
+		secretCache:   secretCache,
+	}
+}
+
+func (r *Manager) NewBackend(registry *mlv1.Registry) (backend.Backend, error) {
+	// Get the secret containing access credentials from llmos-system namespace
+	id, secret, err := getAccessKey(r.secretCache, registry.Spec.S3Config.AccessCredentialSecretName)
+	if err != nil {
+		return nil, fmt.Errorf("get access key failed: %w", err)
+	}
+
+	return s3.NewMinioClient(registry.Spec.S3Config.Endpoint, id, secret, registry.Spec.S3Config.Bucket, registry.Spec.S3Config.UseSSL)
```

golangci-lint (1.64.8)

The line is 132 characters long, which exceeds the maximum of 120 characters.

(lll)